### PR TITLE
feat: add example terraform code

### PR DESCRIPTION
**Pull request by sailorlqh**
Only need to look at the `main.tf` file, the go code change is only for fixing lints, it is already fixed by another PR(https://github.com/abcxyz/jvs-plugin-github/pull/22) which is not merged yet. Will do a rebase after that PR is merged.

### CODE REVIEW

File: terraform/example/main.tf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  # This is the key id under secret manager, where user need to
+  # manully upload the github app private key to.
+  gh_private_key_id="github_app_private_key"
```
Comment (gjonathanhong) [inline terraform/example/main.tf]: please run `terraform fmt`

File: terraform/example/main.tf [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        version: "latest",
+    }
+  }
+}
+
+module "github_plugin" {
+  # Pin to proper version.
+  source = "git::https://github.com/abcxyz/jvs-plugin-github.git//terraform/modules/secret_manager?ref=69fdda2fb914a28e89d352d86a501397f4ddcaad"
+
+  project_id = module.qinhang_jvs_plugin_github_dev.project_id
```
Comment (yolocs) [inline terraform/example/main.tf]: Is this correct?

Comment (sailorlqh) [reply terraform/example/main.tf]: Updated, thanks for catching this.

File: terraform/example/main.tf [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+locals {
+  # This is the key id under secret manager, where user need to
+  # manully upload the github app private key to.
+  gh_private_key_id="github_app_private_key"
+}
+
+module "jvs" {
+  source = "git::https://github.com/abcxyz/jvs.git//terraform/e2e?ref=main" # this should be pinned to the SHA desired
+
+  project_id = "YOUR_PROJECT_ID"
```
Comment (gjonathanhong) [inline terraform/example/main.tf]: nit: can move this to locals since it is referenced multiple times

**Review by gjonathanhong — DISMISSED** [review]

**Review by gjonathanhong — APPROVED** [review]

**Review by sqin2019 — APPROVED** [review]

**Review by capri-xiyue — APPROVED** [review]

**Pull request merged**